robottelo/ui/base.py

```diff
@@ -453,6 +453,23 @@ def is_element_visible(self, locator):
         self.wait_for_ajax()
         return element.is_displayed()
 
+    def element_type(self, locator):
```

Comment:
Goal:
Decide what type element has. Here we have pretty easy idea as we are taking locator (e.g.`<tr class="even">`) and return element type (`tr`) using locator tag name